Formula/php.rb

```diff
@@ -17,26 +17,32 @@ class Php < Formula
   depends_on "argon2"
   depends_on "aspell"
   depends_on "autoconf"
-  depends_on "curl" if MacOS.version < :lion
+  depends_on "brotli"
   depends_on "freetds"
   depends_on "freetype"
   depends_on "gettext"
   depends_on "glib"
   depends_on "gmp"
   depends_on "icu4c"
   depends_on "jpeg"
-  depends_on "libiconv" if DevelopmentTools.clang_build_version >= 1000
   depends_on "libpng"
   depends_on "libpq"
   depends_on "libsodium"
   depends_on "libzip"
-  depends_on "openldap" if DevelopmentTools.clang_build_version >= 1000
+  depends_on "nghttp2"
```

Comment:
Due to the dependency on `boost` we should try and avoid depending on `nghttp2` if possible but see comment below about curl.

Reply:
We could leave it out, but then it wouldn't be feature compatible with what system curl provides. Perhaps in addition to a new formula for `curl-openssl`, we add a formula for `nghttp2-libonly` that only has the `--enable-lib-only` configure flag (no dependencies outside of pkg-config).